tsconfig.json

```diff
@@ -26,14 +26,9 @@
     "test",
     "release",
     "app/main.prod.js",
-    "app/main.prod.js.map",
     "app/renderer.prod.js",
-    "app/renderer.prod.js.map",
-    "app/style.css",
-    "app/style.css.map",
     "app/dist",
     "dll",
-    "app/main.js",
-    "app/main.js.map"
+    "app/main.js"
```

Comment:
Does this file exist for you? I can't find it in my local build

Reply:
nah can't find it, should it be removed too?